src/AdminConsole/Components/Pages/App/Settings/SettingsComponents/ApiKeysSection.razor

```diff
@@ -44,65 +44,31 @@
                             </ul>
                         </td>
                         <td>
-                            <EditForm class="flex space-x-2 ml-0" Model="SelectedForm" FormName="@(SelectedApiKeyFormName)" name="@SelectedApiKeyFormName" OnSubmit="OnSelectedFormSubmitted">
+                            <EditForm confirm-submit="testing" class="flex space-x-2 ml-0" Model="ConfirmedSelectedForm" FormName="@(ConfirmedSelectedApiKeyFormName)" name="@ConfirmedSelectedApiKeyFormName" OnSubmit="OnSelectedFormConfirmed">
                                 @if (apiKey.IsLocked)
                                 {
-                                    <button class="btn-warning flex items-center" name="SelectedForm.Action" value="unlock">
```

Comment:
We can customize the modal by suplying attributes, e.g. "confirm-title="Confirm unlock" etc.